examples/llmcompressor/README.md

```diff
@@ -0,0 +1,73 @@
+
+> [!NOTE]  
+> We also publish compressed versions of popular LLMs to HuggingFace that can be downloaded directly at https://huggingface.co/RedHatAI
+
+## Contents
+
+Two pathways are provided:
+
+1. [A Jupyter Notebook](workbench_example.ipynb) that can be used with the workbench image available at https://quay.io/repository/opendatahub/llmcompressor-workbench.
+
+2. [An example pipeline](oneshot_pipeline.py) splits the notebook up into components that can be run as a Data Science Pipeline, with the runtime image available at https://quay.io/repository/opendatahub/llmcompressor-pipeline-runtime. The goal of this is to highlight how multiple compression algorithms can be compared in parallel, with compressed model artifacts and evaluation results easily shareable with stakeholders in a single web UI.
+
+## Prerequisites
+
+These examples assume the user has access to a Data Science Project on a Red Hat OpenShift AI cluster. Data Science Pipelines must be enabled in your project to run the pipeline. Users must `pip install -r requirements.txt` in order to run `oneshot_pipeline.py`.
+
+### Accessing HuggingFace Hub Models
+
+Many popular open-source models on the HuggingFace Model Hub are [gated](https://huggingface.co/docs/hub/en/models-gated), requiring users to provide basic information and accept a EULA before being granted access to download the model. Users must [generate a token](https://huggingface.co/docs/hub/en/security-tokens) to access models, and provide that token in workbenches and pipelines.
+
+In workbenches, users can embed the token directly in the notebook:
+```python
+import os
+os.environ["HF_TOKEN"] = "<your token here>"
+```
+
+In pipelines, users need to create a secret that can be passed as an env var to the running container. The example pipeline demonstrates this, provided the user has created a secret for the project:
+
+1. In OpenShift console, select Secrets on left sidebar.
+2. Click Create -> Key/value secret with
+    - Secret name: `hf-hub-secret`
+    - Key: `HF_TOKEN`
+    - Value: `<your token here>`
+
+With this, the token will be added whenever the task is modified accordingly:
+
+```python
+kubernetes.use_secret_as_env(
+    calibrated_task,
+    secret_name="hf-hub-secret",
+    secret_key_to_env={"HF_TOKEN": "HF_TOKEN"},
+)
+```
+
+
+### Using Accelerated Hardware
+
+Data-free compression flows can typically run in a short amount of time on CPU, but calibrated compression often require accelerated hardware. The examples demonstrate both, and the images are built to be compatible with NVIDIA GPUs with [compute capability](https://developer.nvidia.com/cuda-gpus) 7.0 or higher.
```

Comment:
Nit: should this be `...often requires accelerated hardware` (requires versus require) ?

Reply:
thanks, updated